Makefile

```diff
@@ -27,8 +27,6 @@ VERSION ?= $(shell cat VERSION)
 REGISTRY ?= digitalocean
 GOVERSION ?= 1.12.3
 
-export GO111MODULE := off
```

Comment:
Rebased on top of https://github.com/digitalocean/digitalocean-cloud-controller-manager/pull/228